app/utils/BFS.js

```diff
@@ -0,0 +1,122 @@
+const pathFind = (field, startP, endP) => {
```

Comment:
1. functions should start with a verb: findPath
1. Don't abbreviate: startPoint, endPoint or pointStart/End. I'm assuming "P" is for "point", but don't know. Could be "path" or "properties" or "pop corn". Don't make the reader guess, ever.
1. Don't use arrow functions unless you need their features: lexical binding and implicit return.
1. I follow the rule that from the third parameter and on, the function should be refactored to a single object argument with destructuring.
Summarizing:
```js
function findPath({ field, startPoint, endPoint, }){
```